modules/javafx.graphics/src/main/native-prism-es2/x11/X11GLFactory.c

```diff
@@ -215,7 +218,26 @@ JNIEXPORT jlong JNICALL Java_com_sun_prism_es2_X11GLFactory_nInitialize
         return 0;
     }
 
-    visualInfo = glXGetVisualFromFBConfig(display, fbConfigList[0]);
+    GLXFBConfig fbConfig;
+    // X Visual of depth = 32 is required for window transparency
+    for (int i = 0; i < numFBConfigs; i++) {
+        XVisualInfo *vi = glXGetVisualFromFBConfig(display, fbConfigList[i]);
+        if (!vi) continue;
+
+        if (vi->depth == 32) {
+            fbConfig = fbConfigList[i];
+            visualInfo = vi;
+            break;
+        }
+
+        XFree(vi);
+    }
+
+    if (visualInfo == NULL) {
+        fbConfig = fbConfigList[0];
```

Comment:
If I understand this correctly, we're just fetching the first available option from the list which might not always have windows transparency. If that's the case - would it make sense to print some sort of warning on the terminal that window transparency might not work correctly? I think it could be a valuable information to the user, or at least when debugging in the future.

@kevinrushforth it's also a question to you, cause I'm not 100% sure what the JFX strategy would be in this case - if we would want this to be printed always as a warning, or only when we ex. activate `-Dprism.verbose=true`